### PR TITLE
Add denali versions

### PR DESCRIPTION
**Pull request by badrogger**

### CODE REVIEW

File: validators/versions-denali.md [hunk 1]
```diff
@@ -4,41 +4,75 @@ You can find the compatible versions in this page. Set up requires only Validato
...
+
+
 ### Mainnet Versions for Set up
 
-**Node CLI version**: [1.0.0](https://github.com/skalenetwork/skale-node-cli/releases/download/1.0.0/skale-1.0.0-Linux-x86_64) 
+**Node CLI**: [1.1.0](https://github.com/skalenetwork/node-cli/releases/download/1.1.0/skale-1.1.0-Linux-x86_64) 
+
+**Validator CLI**:  [1.2.0](https://github.com/skalenetwork/validator-cli/releases/tag/1.2.0)
+
+**SGX Wallet**: [WIP](https://github.com/skalenetwork/sgxwallet/releases/tag/1.58.5-stable.1)
```
Comment (ebruengwall-ant) [inline validators/versions-denali.md]: this looks old

File: validators/versions-denali.md [hunk 2]
```diff
@@ -4,41 +4,75 @@ You can find the compatible versions in this page. Set up requires only Validato
...
 
-**SKALE Manager version**: [1.5.2](https://github.com/skalenetwork/skale-network/tree/master/releases/mainnet/skale-manager/1.5.2)
+**SKALE Manager version**: [1.7.2](https://github.com/skalenetwork/skale-network/tree/master/releases/mainnet/skale-manager/1.7.2)
+
+**Node CLI**: [1.0.1](https://github.com/skalenetwork/node-cli/releases/download/1.0.1/skale-1.0.1-Linux-x86_64) 
+
+**Validator CLI**:  [1.1.2](https://github.com/skalenetwork/validator-cli/releases/tag/1.1.2)
 
-**Skaled version**: [1.46-stable.0](https://github.com/skalenetwork/skaled/releases/tag/1.46-stable.0)
+**Skaled version**: [1.58.5-stable.1](https://github.com/skalenetwork/skaled/releases/tag/1.58.5-stable.1)
```
[the remark this answers is not in the document]
Comment (badrogger) [reply validators/versions-denali.md]: I thought `Mainnet versions for review` - current mainnet versions (compatible with 1.7.2 skale-manager). Am I wrong ?

File: validators/versions-denali.md [hunk 3]
```diff
@@ -4,41 +4,75 @@ You can find the compatible versions in this page. Set up requires only Validato
...
 **watchdog**: [1.0.0-stable.0](https://github.com/skalenetwork/skale-watchdog/releases/tag/1.0.0-stable.0)
 
-#### Mainnet Environment Variables
+#### Mainnet Environment Variables to set
 
 ```shell
 MONITORING_CONTAINERS=True
-DOCKER_LVMPY_STREAM=1.0.0
-MANAGER_CONTRACTS_ABI_URL=https://raw.githubusercontent.com/skalenetwork/skale-network/master/releases/mainnet/skale-manager/1.5.2/skale-manager-1.5.2-mainnet-abi.json
+DOCKER_LVMPY_STREAM=1.0.1-stable.1
```
[the remark this answers is not in the document]
Comment (gannakulikova) [reply validators/versions-denali.md:68]: @badrogger @ebruoguzberk  correct version is 1.0.1-stable.1
seems that D1 did not update the version on Confluence

File: validators/versions-denali.md (hunk 1 above)
Comment (gannakulikova) [reply validators/versions-denali.md]: @ebruoguzberk @badrogger  we're still waiting for the final version of SGX wallet, we can put 1.70.0-stable.0 for now and update if needed

**Review by ebruengwall-ant — APPROVED** [review]

**Pull request merged**